ui/scss/component/_main.scss

```diff
@@ -115,7 +115,7 @@
 
 .main__document-wrapper--markdown {
   @extend .main__document-wrapper;
-  width: 40em;
+  width: 60em;
```

Comment:
I'm in favor of keeping this small. Most things I've read recommend 50 - 70 characters per line for maximum readability. We are already over that with 40em. The latest lbrytech blog post has 83 characters for the first word right now. With these changes, that is increased to 107 characters.